sigs/sig-scalability/README.md

```diff
@@ -0,0 +1,23 @@
+# Argo SIG Scalability
+
+Argo SIG Scalability is focused on the scalability and performance of the Argo projects.
+
+The initial focus of the group is on Argo CD, working through the [the Argo CD Scalability Benchmarking proposal](https://github.com/argoproj/argo-cd/pull/12662). When the time comes that other's want to join and focus on another project (e.g. Argo Workflows), [a proposal](https://github.com/argoproj/argoproj/tree/master/proposals) should be created to expand the scope of the SIG.
```

Comment:
I think this can be handled within the SIG without the need for a new separate proposal, by giving the SIG a bit more loose charter. Doesnt mean that other areas will be covered now (or ever) but reduces some process.